tests/goth_tests/test_mid_agreement_payments/requestor.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+import asyncio
+import logging
+from typing import AsyncIterable
+from datetime import timedelta
+
+from yapapi import Golem, Task, WorkContext
+from yapapi.log import enable_default_logger, log_event_repr
+from yapapi.payload import vm
+from yapapi.props import com
+from yapapi.props.builder import DemandBuilder
+from yapapi.strategy import (
+    LeastExpensiveLinearPayuMS,
+    PROP_DEBIT_NOTE_INTERVAL_SEC,
+    PROP_PAYMENT_TIMEOUT_SEC,
+    PropValueRange,
+    MarketStrategy,
+    SCORE_TRUSTED,
+)
+
+
+class ShortDebitNoteIntervalAndPaymentTimeout(MarketStrategy):
+    acceptable_prop_value_range_overrides = {
+        PROP_DEBIT_NOTE_INTERVAL_SEC: PropValueRange(15, 20),
+        PROP_PAYMENT_TIMEOUT_SEC: PropValueRange(20, 25),
+    }
+
+    async def decorate_demand(self, demand: DemandBuilder) -> None:
+        await super().decorate_demand(demand)
+        demand.ensure(f"({com.PRICE_MODEL}={com.PriceModel.LINEAR.value})")
```

Comment:
you could have just inherited from `LeastExpensiveLinearPayuMS` ...

Reply:
Fixed. BTW: I even already imported `LeastExpensiveLinearPayuMS`. :)